<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Converter .rtf</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Открыт:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCurrentFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="sourceText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonConvertOnce">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Конвертировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPrev">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Предыдущий</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Директория сохранения:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCurrentDirectorySave">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>./</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="convertedText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonNext">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Следующий</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonConvertAll">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Конвертировать все</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSimpleText">
        <property name="text">
         <string>Конвертировать в текст без тегов</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIsRegex">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Использовать regex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSelectPath"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuThemes">
    <property name="title">
     <string>Тема</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuThemes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSelectPath">
   <property name="text">
    <string>Папка сохранения</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>fileOpen(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectPath</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>selectPath(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNext</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>buttonPreview(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPrev</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>buttonPreview(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConvertOnce</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>convertOnce(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConvertAll</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>convertAll(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileOpen(bool)</slot>
  <slot>selectPath(bool)</slot>
  <slot>buttonPreview(bool)</slot>
  <slot>convertOnce(bool)</slot>
  <slot>convertAll(bool)</slot>
 </slots>
</ui>
